distribution/src/main/assembly/dist.xml

```diff
@@ -85,11 +90,17 @@
                 <exclude>**/zanata.sh</exclude>
                 <exclude>**/zanata.xml</exclude>
                 <exclude>src/main/scripts/**</exclude>
+                <exclude>hibernate-noorm-release-scripts/**</exclude>
 
                 <!-- actual files which should be ignored -->
+                <exclude>.travis.yml</exclude>
+                <exclude>travis/**</exclude>
                 <exclude>.git</exclude>
                 <exclude>.gitignore</exclude>
                 <exclude>**/target/**</exclude>
```

Comment:
Good thinking; we had the same concern in MapStruct and added a [CS rule](https://github.com/mapstruct/mapstruct/blob/master/build-config/src/main/resources/build-config/checkstyle.xml#L55) to avoid such package names. I'll do the same here and merge.